shell/platform/android/io/flutter/embedding/engine/loader/ResourceExtractor.java

```diff
@@ -42,6 +42,14 @@
   @NonNull private final HashSet<String> mResources;
   private FutureTask mExtractTask;
 
+  // Thread pool settings are the same as default values from the deprecated android.os.AsyncTask
+  // API, which was previously used (see
+  // https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/os/AsyncTask.java#211).
```

Comment:
nit: this link might not be valid in future because it doesn't point to a revision. 

However, if you add a Java doc with {@link android.os.AsyncTask}, then cross-reference just works out of the box in Android Studio or CS (code search).